### ISSUE TITLE
Missing artifacts in AppVeyor

### ISSUE THREAD

**Issue by NN---** [notepad-plus-plus/notepad-plus-plus#1985]
https://ci.appveyor.com/project/donho/notepad-plus-plus/build/1.0.133#L1677

```
Error pushing artifact: File not found: C:\projects\notepad-plus-plus\PowerEditor\bin
```

In artifacts there is only notepad++.exe ( https://ci.appveyor.com/project/donho/notepad-plus-plus/build/1.0.133/artifacts )

**Comment by dail8859** [comment]
I don't know enough about AppVeyor but this may be related to #1942 with a fix provided in #1943.

**Comment by cmeriaux** [comment]
Hello
I confirm, I fixed this problem in #1942  #1943 
@NN---  Can you close it ?
